internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -98,10 +94,10 @@ func NewLoadBalancedWebService(conf LoadBalancedWebServiceConfigs,
 			AccountPrincipalARN: conf.RootUserARN,
 		}
 		httpsEnabled = true
-	} else {
-		if conf.Env.HasImportedCerts() {
-			httpsEnabled = true
-		}
+	}
+	if conf.Env.HasImportedCerts() {
+		httpsEnabled = true
```

Comment:
could it happen that env has imported cert but ALB is disabled by `http: false` 🤔